.drone.star

```diff
@@ -60,32 +60,11 @@ def testing(ctx):
           },
         ],
       },
-      {
-        'name': 'staticcheck',
-        'image': 'webhippie/golang:1.13',
-        'pull': 'always',
-        'commands': [
-          'make staticcheck',
-        ],
-        'volumes': [
-          {
-            'name': 'gopath',
-            'path': '/srv/app',
-          },
-        ],
-      },
```

Comment:
ok, seems interesting to change that in the boilerplate

Reply:
@micbar issue on ocis-boilr?

Reply:
perhaps I have to review my boilr local version, it might be outdated :)